src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -1407,6 +1412,26 @@ private void sendChangeGameStatePacket(@NotNull ChangeGameStatePacket.Reason rea
         playerConnection.sendPacket(changeGameStatePacket);
     }
 
+    /**
+     * Gets the TabList the player is viewing
+     * Will be null if the {@link net.minestom.server.tab.TabListPopulator} has not assigned a TabList
+     *
+     * @return the viewed {@link TabList}
+     */
+    public @Nullable TabList getTabList() {
```

Comment:
Should be `NotNull`, as the default `TabList` should be used (showing all online players)